--- dmenu.c	2013-11-26 09:15:04.000000000 +0100
+++ dmenu_patched.c	2017-07-14 22:28:44.975939402 +0200
@@ -331,7 +331,8 @@ drawmenu(void) {
     if(!quiet || strlen(text) > 0) {    
         if(lines > 0) {
             /* draw vertical list */
-            dc->w = mw - dc->x;
+            dc->x = 0;
+            dc->w = mw;
             for(item = curr; item != next; item = item->right) {
                 dc->y += dc->h;
                 drawtext(dc, item->text, (item == sel) ? selcol : normcol);
